x-pack/plugins/endpoint/common/types.ts

```diff
@@ -647,9 +647,16 @@ interface HostPolicyResponseActions {
   [key: string]: HostPolicyResponseActionDetails;
 }
 
+export interface HostPolicyResponseConfiguration {
```

Comment:
(optional) Maybe leave these in the type below `HostPolicyResponse` so that we can get a single view into the Policy type in one place and instead define this new type by referencing this type in `HostPolicyResponse`:

```typescript
export type HostPolicyResonseConfiguration = HostPolicyResponse['endpoint']['policy']['applied']['response']
```